src/jc21/CliTable.php

```diff
@@ -531,6 +531,21 @@ protected function getColorFromName($colorName)
     }
 
 
+    /**
+     * getValue
+     *
+     * @access protected
+     * @param  (array | ArrayAccess)  $row
+     * @param  string  $key
+     * @param  any  $rowKey
+     * @return string
+     */
+    protected function getValue($row, string $key, $rowKey)
+    {
+        return $row[$key];
```

Comment:
According to this `getValue` method, it looks like the `$rowKey` parameter is not used.

Please remove this.

Reply:
I think that might be on purpose, as with extended classes the function signature needs to be the same in php 7+ versions (can't remember which one specifically)

Reply:
Hello, 

Yes that was the point : check the method signature and see what you will receive as args if you override the method.